tests/manage/ocs_288.py

```diff
@@ -0,0 +1,407 @@
+    out = ocp.exec_ceph_cmd(cmd)
+    for item in out:
+        if item['poolname'] == pool_name:
+            ceph_validate = True
+    pool_obj = POOL.get()
+    sample = pool_obj['items']
+    for item in sample:
+        if item.metadata.name == pool_name:
+            k8s_validate = True
+    if ceph_validate and k8s_validate:
+        return True
+    else:
+        if not ceph_validate:
+            log.error(f"{pool_name} pool wasn't created at CEPH side")
+        if not k8s_validate:
+            log.error(f"{pool_name} pool wasn't created at K8S side")
+        return False
+
+
+def validate_storageclass(sc_name):
+    """
+    Validate if storageClass is been created or not
+    """
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+    for item in sample:
+        if item.metadata.name in sc_name:
+            log.info(f"StorageClass got created")
+            return True
+    return False
+
+
+def create_cephfilesystem(kwargs):
+    """
+    Function for deploying CephFileSystem (MDS)
+    :param kwargs:
+    :return:
+    """
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        CEPH_FILESYSTEM, **kwargs
+    )
+    with open(TEMP_YAML_FILE_FS, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new CephFileSystem")
+    assert CFS.create(yaml_file=TEMP_YAML_FILE_FS)
+    fs_name = file_y.get('metadata')['name']
+    fs_name = fs_name
+    POD.wait_for_resource(condition="Running", selector='app=rook-ceph-mds')
+    assert validate_cephfilesystem(fs_name=fs_name)
+    return True
+
+
+def validate_cephfilesystem(fs_name):
+    """
+    Function for validating CephFileSystem Creation
+    :param fs_name:
+    :return:
+    """
+    ceph_validate = False
+    k8s_validate = False
+    cmd = "ceph fs ls"
+    out = ocp.exec_ceph_cmd(cmd)
+    if out:
+        out = out[0]['name']
+        if out == fs_name:
+            log.info("FileSystem got created from Ceph Side")
+            ceph_validate = True
+        else:
+            log.error("FileSystem was not present at Ceph Side")
+            return False
+    result = CFS.get()
+    sample = result['items']
+    for item in sample:
+        if item.metadata.name == fs_name:
+            log.info(f"Filesystem got created from kubernetes Side")
+            k8s_validate = True
+        else:
+            log.error("Filesystem was not create at Kubernetes Side")
+            return False
+    if ceph_validate and k8s_validate:
+        return True
+    else:
+        return False
+
+
+def create_storageclass_rbd(kwargs):
+    """
+    Create a new CSI StorageClass
+    """
+    pool_name = kwargs.get('rbd_pool')
+    sc_name = kwargs.get('rbd_storageclass_name')
+    assert create_rbd_pool(pool_name=pool_name)
+    assert validate_pool_creation(pool_name)
+    if validate_pool_creation(pool_name):
+        log.info("Resource validation Passed")
+    else:
+        return 1
```

Comment:
In line 280 you are returning a bool and here int.
What is the right returning type?
Please add a description to this function with an explanation of the return type

Reply:
Done